pages/03.Tutorials/10.upload-images/default.en.md

```diff
@@ -1,16 +1,20 @@
 ---
-title: 'Upload custom images'
+title: 'Upload images'
 published: true
-date: '08-08-2018 11:34'
+date: '17-08-2018 16:26'
 taxonomy:
     category:
         - tutorial
 ---
 
-## Uploading Custom Images
+## Upload Images
 
 There are three different ways to upload custom images.
 
+### Image sources
+
+!!! [This overview](../../02.Documentation/07.images/default.en.md) contains a list of standard images sources.
```

Comment:
this section directly under the phrase above might be disappointing. perhaps move it to the end?

…a list of sources for commonly used images.